<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1360</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgba(32, 32, 32,64);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCheckBox" name="cbAF34">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>10</y>
      <width>101</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0) ;
background-color: rgba(255, 0, 0, 32) ;
border-radius:6px;</string>
    </property>
    <property name="text">
     <string>AF3 - AF4</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbF34">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>60</y>
      <width>101</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 80, 0) ;
background-color: rgba(255, 80, 0, 32) ;
border-radius:6px;</string>
    </property>
    <property name="text">
     <string>F3 - F4</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbF78">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>110</y>
      <width>101</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 0) ;
background-color: rgba(255, 255, 0, 32) ;
border-radius:6px;</string>
    </property>
    <property name="text">
     <string>F7 - F8</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbFC56">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>160</y>
      <width>101</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 255, 0) ;
background-color: rgba(0, 255, 0, 32) ;
border-radius:6px;</string>
    </property>
    <property name="text">
     <string>FC5 - FC6</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbT78">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>210</y>
      <width>101</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 255, 255) ;
background-color: rgba(0, 255, 255, 32) ;
border-radius:6px;</string>
    </property>
    <property name="text">
     <string>T7 - T8</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbP78">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>260</y>
      <width>101</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 255) ;
background-color: rgba(0, 0, 255, 32) ;
border-radius:6px;</string>
    </property>
    <property name="text">
     <string>P7 - P8</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbO12">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>310</y>
      <width>101</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 255) ;
background-color: rgba(255, 0, 255, 32) ;
border-radius:6px;</string>
    </property>
    <property name="text">
     <string>O1 - O2</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbAll">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>370</y>
      <width>91</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(200, 200, 200) ;
background-color: rgba(200, 200, 200, 32) ;
border-radius:6px;</string>
    </property>
    <property name="text">
     <string>ALL</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressDelta">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>400</y>
      <width>101</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 0);
background-color: rgba(255, 255, 0, 32);
border-radius:6px ;</string>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressTheta">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>400</y>
      <width>101</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 80, 0);
background-color : rgba(255, 80, 0, 32);
border-radius:6px ;</string>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressAlpha">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>400</y>
      <width>101</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 255);
background-color : rgba(0, 0, 255, 32);
border-radius:6px ;</string>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBeta">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>400</y>
      <width>101</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 255, 0);
background-color : rgba(0, 255, 0, 32);
border-radius:6px ;</string>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressGamma">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>400</y>
      <width>101</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
background-color : rgba(255, 0, 0, 32);
border-radius:6px ;</string>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelDelta">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>630</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 0);
background-color: rgba(255, 255, 0, 32);
border-radius:6px ;</string>
    </property>
    <property name="text">
     <string>DELTA</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelTheta">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>630</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 80, 0);
background-color : rgba(255, 80, 0, 32);
border-radius:6px ;</string>
    </property>
    <property name="text">
     <string>THETA</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelAlpha">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>630</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 255);
background-color : rgba(0, 0, 255, 32);
border-radius:6px ;</string>
    </property>
    <property name="text">
     <string>ALPHA</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelBeta">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>630</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 255, 0);
background-color : rgba(0, 255, 0, 32);
border-radius:6px ;</string>
    </property>
    <property name="text">
     <string>BETA</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelGamma">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>630</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
background-color : rgba(255, 0, 0, 32);
border-radius:6px ;</string>
    </property>
    <property name="text">
     <string>GAMMA</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphPosition">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>410</y>
      <width>361</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(../epocWatch/images/head.png);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbConnect">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>450</y>
      <width>80</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255) ;
background-color:rgba(0, 0, 255, 32) ;
border-radius:12px ;</string>
    </property>
    <property name="text">
     <string>CONNECT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbStart">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>450</y>
      <width>80</width>
      <height>64</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255) ;
background-color:rgba(0, 255, 0, 32) ;
border-radius:12px ;</string>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbQuit">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>530</y>
      <width>120</width>
      <height>110</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255) ;
background-color:rgba(255, 0, 0, 32) ;
border-radius:12px ;</string>
    </property>
    <property name="text">
     <string>QUIT</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphSignal">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>1060</width>
      <height>370</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(64, 64, 64, 64);</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1360</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 0, 0) ;</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
